index.html

```diff
@@ -151,6 +151,16 @@ <h2>Data Model</h2>
             property, and <code>JsonSchemaCredential2023</code> for usage of a [[JSON-Schema]] represented as a
             <a>verifiable credential</a>.
          </p>
+         <p>
+            Implementers may find use in packaging a JSON Schema as a verifiable credential when they wish to
+            leverage features of the [[VC-DATA-MODEL-2]], answering questions such as:
+            <ul>
+               <li>Who is the author of this schema? (provided by the <code>issuer</code> property)</li>
+               <li>Why should the author be trusted for providing this schema? (provided by the <code>evidence</code> property)</li>
```

Comment:
Do you have an example of using evidence for this? 
If not, please remove the evidence link.

Reply:
removed